### ISSUE TITLE
Mod Suggestions/Requests

### ISSUE THREAD

**Issue by MCaesar** [thraaawn/CompactMachines#75]
Love the mod, a few ideas that would be nice if they're not too difficult to implement:
1. Block/room security: A way to set user access to a room to prevent people messing with things inside, and prevent them from breaking the block itself (ruining the build you have inside).
2. Block/room relocation: Allow someone to move a room and keep the contents inside the same at the new location, would allow for mobile setups, and wouldn't have to redo a room if the block needs to be moved.
3. Room extension: Instead of a quantum entangler accessing the same room from 2 separate blocks, allow for the 2 blocks to be added together to create a larger room. In my current case, I have 2 things that I'd like to put in a compact machine that won't fit inside a max size room (big reactors turbine[20x7x7] setup and tier 6 altar from Blood Magic[23x23x9])
4. Multiple input/output from the same side: Mainly for use with EnderIO conduits that allow item/fluid/energy/redstone in/out of 1 port.
5. Change location of port: Sometimes a setup could benefit if you could change the location of the input/output port inside the room to somewhere other than the exact middle of the wall.

**Comment by thraaawn** [comment]
1. Is kind of planned already. Not sure whether I will actually implement it. Compact Machines are "compact machines" and not "unbreakable safes". I think it's more interesting to alleviate some system to protect your base than to have protection built right in.
2. Already possible. Upgrade the machine and :cake:
3. Never going to happen as it breaks the concept of "be as efficient as possible". It would require a single CM to occupy more than one chunk. There might be machines *higher* than the base-dimension, but the sides are and always will be limited. Don't expect anything here either.
4. Already possible and working fine.
5. Already planned and will probably be in the next release.

tl;dr;
1+3. Maybe, but unlikely to happen.
2+4. Already possible
5. Planned, next release.

Thanks for your feedback!
